includes/class-wi-volunteer-management.php

```diff
@@ -81,6 +81,7 @@ public function __construct() {
 
 		// Load the public hooks
 		$this->define_public_hooks();
+      $this->define_widget_hooks();
```

Comment:
Sorry, I think I wasn't clear enough on this one. I really like that you put the widget itself in its own file. I was really just referring to the need for `define_widget_hooks()`. I didn't think we needed a separate function for declaring widget hooks. 

That said, you're welcome to leave it as is and we can update it on our end.